src/components/tooltip/Tooltip.js

```diff
@@ -7,7 +7,7 @@ import * as React from 'react';
 import ReactDOM from 'react-dom';
 import type { CssPixels } from 'firefox-profiler/types';
 
-import { ensureExists } from 'firefox-profiler/utils/flow';
+import { ensureExists } from '../../utils/flow';
```

Comment:
Be careful, in some conflict fixing you changed this coming from a previous patch.

Reply:
Yeah, that was an error from my part. Didn't see it until I pushed my changes.